### DISCUSSION TITLE
TS0504B RGBW led strip controller

### DISCUSSION THREAD

**Discussion by JimmiJonathan** [Koenkk/zigbee2mqtt#17049] [Devices]
Hello, 

Is there a way to control by this controller more than one strip ? I need to control 2 or 3 white led strips. In Z2M I cannot find option to control separetly colors of rgbw strip...

![image](https://user-images.githubusercontent.com/82450195/225303240-d04195e4-e1e9-4692-a8f9-7d1f625861ab.png)

I think its this product: https://pl.aliexpress.com/item/725319185.html?spm=a2g0o.productlist.main.11.c0e97f00M0sVPI&algo_pvid=60fe73e6-f65d-469d-bf09-2688fc5ba35f&aem_p4p_detail=2023031505035311616097869499350000850224&algo_exp_id=60fe73e6-f65d-469d-bf09-2688fc5ba35f-5&pdp_ext_f=%7B%22sku_id%22%3A%2212000021179293231%22%7D&pdp_npi=3%40dis%21PLN%2163.93%2144.74%21%21%21%21%21%402100bbf516788818338994288d06f5%2112000021179293231%21sea%21PL%21767752927&curPageLogUid=hWeavISQIWFD&ad_pvid=2023031505035311616097869499350000850224_6&ad_pvid=2023031505035311616097869499350000850224_6

I bought this from polish site:
https://www.houseiq.pl/pl/p/Kontroler-LED-RGB-15A-ZigBee-3.0-HUE-TUYA/1796

Maybe you have a way to modify it ? :)
Thanks !

**Comment by michalholes** [comment]
Hello, did you found some solution? I want to control 2 strips too (RGB+W), but Z2M can handle only one :( When I turn on some colour, Z2M  shuts down white and vice versa :(

**Comment by JimmiJonathan** [comment]
Unfortunately not. I change to shelly rgbw and this is working
